packages/bpk-component-card-list/src/BpkCardListGridStack/BpkCardListGridStack.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState } from 'react';
+
+import { BpkButtonV2 } from '../../../bpk-component-button';
+import { cssModules } from '../../../bpk-react-utils';
+import BpkExpand from '../BpkExpand';
+import { ACCESSORY_TYPES, type CardListGridStackProps } from '../common-types';
+
+import STYLES from './BpkCardListGridStack.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkCardListGridStack = (props: CardListGridStackProps) => {
+  const {
+    accessory,
+    buttonText,
+    children,
+    expandText,
+    initiallyShownCards,
+    layout,
+    onButtonClick,
+  } = props;
+
+  let defaultInitiallyShownCards: number;
+  if (accessory === ACCESSORY_TYPES.Expand) {
```

Comment:
I don't think this depends on the accessory. From the API table:
> 3 cards displayed by default, but you can change it.

Reply:
This is for the case when more than 3 cards are given but the user don't want to use an expand accessory. It would look confusing for the user as it would simply look like the component is broken with only 3 cards showing